onnxruntime/python/tools/profile_explorer/profile_explorer.py

```diff
@@ -47,6 +50,28 @@ def _get_args():
     parser.add_argument("--csv", help="save data to csv")
     parser.add_argument("-c", "--count", type=int, default=40, help="list top N items")
     parser.add_argument("-v", "--verbose", action="store_true", help="verbose")
+
+    parser.add_argument(
+        "--start",
+        "-s",
+        type=int,
+        default=1,
+        help="index of the first model run to process",
```

Comment:
Please clarify whether this expects a 0-based index or a 1-based index. I suspect it's 1-based, considering the default value, but it would be good to document it.

Reply:
Updated. It is 0-based, defaults to 1 to exclude run 0.